__tests__/app.test.js

```diff
@@ -154,4 +154,53 @@ describe('GET comments', () => {
                 expect(comments).toEqual([])
             })
     })
+})
+describe('POST 201 create comment for article', () => {
+    test('creates new comment for passed article_id and responds with that comment', () => {
+        const newComment = { username: "rogersop", body: "This is a test comment" }
+        const insertedComment = { author: "rogersop", body: "This is a test comment", votes: 0, article_id: 10 }
+        return request(app)
+            .post('/api/articles/10/comments').send(newComment)
+            .expect(201)
+            .then(({ body }) => {
+                const { comment } = body
+                expect(comment).toMatchObject(insertedComment)
+            })
+    })
+    test('POST 400 if passed comment does not have username property', () => {
+        const newComment = { user: "rogersop", body: "This is a test comment" }
+        return request(app)
+        .post('/api/articles/10/comments').send(newComment)
+        .expect(400)
+        .then(({ body }) => {
+            expect(body.msg).toBe('Invalid input (not_null_violation)')
+        })
```

Comment:
Ideally, we want our error messages to be generic. Being overly specific error messages gives others an idea of how to access specific information in your database 🔍